plugins/embark-assistant/finder_ui.cpp

```diff
@@ -1041,15 +1267,19 @@ namespace embark_assist {
             Screen::drawBorder("Embark Assistant Site Finder");
 
             embark_assist::screen::paintString(lr_pen, 1, 1, "4/6");
-            embark_assist::screen::paintString(white_pen, 4, 1, ":Shift list");
-            embark_assist::screen::paintString(lr_pen, 16, 1, "8/2");
-            embark_assist::screen::paintString(white_pen, 19, 1, ":Up/down");
-            embark_assist::screen::paintString(lr_pen, 28, 1, "ENTER");
-            embark_assist::screen::paintString(white_pen, 33, 1, ":Select item");
-            embark_assist::screen::paintString(lr_pen, 46, 1, "f");
-            embark_assist::screen::paintString(white_pen, 47, 1, ":Find");
-            embark_assist::screen::paintString(lr_pen, 53, 1, "ESC");
-            embark_assist::screen::paintString(white_pen, 56, 1, ":Abort");
+            embark_assist::screen::paintString(white_pen, 4, 1, ":<->");
```

Comment:
Could use `"\x1b\x1a"` for actual arrows - see http://dwarffortresswiki.org/index.php/CP437

Reply:
I'll try that.

Reply:
Good suggestion.  I also tried up/down arrows, but Phoebus uses a spruce like thingie instead of the up arrow, so I went back there.